test/common/api_utils.py

```diff
@@ -0,0 +1,170 @@
+from on_http_api2_0 import ApiApi as Api
+from config.api2_0_config import config
+from json import loads
+import sys
+import os
+
+
+def api_node_select_argsList(nodeType='compute', validate_obm=False):
+    """
+    This routine produces a list of node Id's that satisfy the requested parameters. 
+    Nodes may be filtered by Node Id,  SKU name, OBM MAC address which are defined
+    in the system environment.  The node type and OBM setting may be specified.
+    :param  nodeType: Node type (i.e. compute) (default= 'compute')
+    :param  validate_obm: should the node have OBM settings (default=False)
+    return: A list with node IDs that match node type, SKU, and possible valid OBM settings.
+    """
+    return api_node_select(config.api_client,
+                           nodeId=os.getenv("NODEID", "None"),
+                           skuName=os.getenv("SKU", "all"),
+                           obmmac=os.getenv("OBMMAC", "all"),
+                           nodeType=nodeType,
+                           validate_obm=validate_obm,
+                           verbosity=os.getenv("VERBOSITY", "0")
+                           )
+
+
+
+def api_node_select(client,
+                    nodeId=None,
+                    skuName='all',
+                    obmmac='all',
+                    nodeType='compute',
+                    validate_obm=False,
+                    verbosity=None
+                    ):
```

Comment:
one more problem is the naming for each argument. sometimes the words in argument is connected with underscore, such as `validate_obm`, sometimes they are connected by capitalizing the first letter, such as `skuName`, and sometime you connect them with all lower letters, such as `obmmac`.
Keep the variable naming in the same style is important for readability.  As I know,  the python prefer the underscore version.